data/korath/korath.txt

```diff
@@ -333,6 +342,17 @@ fleet "Korath Defense"
 		"'olofez" 4
 		"'olofez (Expeller)" 2
 
+fleet "Korath Surveillance"
+	government "Korath"
+	names "korath"
+	fighters "korath probes"
+	cargo 1
```

Comment:
Perhaps add a comment above the definition that this fleet is not intended for use near systems with planets with Remnant outfitters.